packages/infra/lib/api-stack/api-service.ts

```diff
@@ -236,6 +247,40 @@ export function createAPIService({
     ),
   });
 
+  const nlb = new NetworkLoadBalancer(stack, `ApiNetworkLoadBalancer`, {
```

Comment:
do we need to add explicit permissions for nlb to talk to alb
Like add security groups and an ingressRule? 
Or is that handled implicitly

Reply:
AFAIK its handled implicitly - that's how we did for IHE GW v1.